corehq/apps/domain/forms.py

```diff
@@ -1322,17 +1323,14 @@ def clean_email(self):
             return self.cleaned_data['email']
 
 
-class HQSetPasswordForm(SetPasswordForm):
+class HQSetPasswordForm(EncodedPasswordChangeForm, SetPasswordForm):
```

Comment:
Would need do that for various forms then.. one form would not work for all different forms.
Making this as a `mixin` would make it clear?